sapphire/analysis/reconstructions.py

```diff
@@ -447,3 +441,17 @@ def _store_reconstruction(self, coincidence, core_x, core_y, theta, phi,
             row['s%d' % number] = True
 
         row.append()
+
+    def _get_active_stations(self):
+        """Return station numbers with non-empty event table in datafile"""
+        def stations_with_events():
```

Comment:
The number of lines is exactly the same, I find the list more readable (hey wait, why another function?), but is the generator really faster? If true, I'll have to try and wrap my head around that.